lib/tests/streamlit/elements/data_editor_test.py

```diff
@@ -938,3 +938,16 @@ def test_shows_cached_widget_replay_warning(self):
         el = self.get_delta_from_queue(-2).new_element.exception
```

Comment:
Looks like there's a test that we previously had for the old behavior (on L914) that we'll want to correct here (I can't comment on it directly since it's not part of the diff).